src/SprykerEco/Zed/Heidelpay/Business/Payment/Transaction/TransactionLogReader.php

```diff
@@ -87,6 +87,70 @@ public function findOrderAuthorizeTransactionLogByIdSalesOrder(int $idSalesOrder
         return $this->buildTransactionTransfer($spyTransactionLog);
     }
 
+    /**
+     * @param int $idSalesOrder
+     *
+     * @return \Generated\Shared\Transfer\HeidelpayTransactionLogTransfer|null
+     */
+    public function findOrderAuthorizeOnRegistrationTransactionLogByIdSalesOrder($idSalesOrder)
+    {
+        $spyTransactionLog = $this->findOrderAuthorizeOnRegistrationTransactionEntity($idSalesOrder);
```

Comment:
First of all - no entities in the business layer!
And you should call entity variables as $someNameEntity

Reply:
Skipped (@alex-galych ).